src/components/Home.jsx

```diff
@@ -7,7 +7,7 @@ const Home = () => {
 
   let history = useHistory();
 
-  const Token = function generateToken() {
+  const generateToken = () => {
     const token = getToken();
```

Comment:
I really like your clear function names, it makes the code much more readable :)